tools/renew-certificates.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+gateway_dir="/home/pi/mozilla-iot/gateway"
+
+certbot renew \
+    --config-dir "$gateway_dir/etc" \
+    --logs-dir "$gateway_dir/var/log" \
```

Comment:
I'd prefer to not top-level `etc` and `var` directories into the gateway if we don't have to. Alternatively, they could be added to .gitignore